homeassistant/components/template/sensor.py

```diff
@@ -1,6 +1,12 @@
-"""Allows the creation of a sensor that breaks out state_attributes."""
+"""
+Allows the creation of a sensor that breaks out state_attributes.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.template/
```

Comment:
We're not adding the link in the docstring anymore.

Please revert this change.

Reply:
Have removed this